docs/partner-onboarding/onboarding-overview/incentive-management.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Incentive Management
+order: 1
+---
+
+# Introduction
+The Balancer ecosystem is utilizing a modified version of Curve's vyper gauge infrastructure. Here, we outline how you can utilize our staking gauge system. We cover both how you can apply for a veBAL gauge to receive BAL rewards and how secondary reward programs can be set up.
+
+## BAL Incentives through veBAL Gauges
+BAL is emitted to whitelisted staking gauges in our veBAL system. For a pool to be eligible for BAL rewards, it needs to be voted in by governance. The process on how to apply for a BAL staking gauge can be found [here](../balancer-v2/gauge-onboarding.md). A guide on how to set up a gauge can be found in the gauge creation [section](incentive-management.md#gauge-creation)
+
+## Secondary Reward Token Incentives
+The Balancer Maxis have built a sophisticated infrastructure to create and manage secondary reward campaigns for Balancer staking gauges. To make full use of this system, the Maxis provide tooling to facilitate the setup.
```

Comment:
Could potentially add a separate callout that tells me that all? below steps can be managed on the operations UI (small explainer of why it is named that way?) and what the "operations UI" is. While I know that operations UI and balancer.defilytica.tools is the same (due to the presentation last week) it might not be that obvious to the reader.